tests/utils/test_search_utils.py

```diff
@@ -577,3 +579,46 @@ def test_pagination(page_token, max_results, matching_runs, expected_next_page_t
 def test_invalid_page_tokens(page_token, error_message):
     with pytest.raises(MlflowException, match=error_message):
         SearchUtils.paginate([], page_token, 1)
+
+
+
+def test_single_run_by_ID(run_id, returned_id):
```

Comment:
Let's remove this and add tests in `tests/store/tracking/test_file_store.py` and `tests/store/tracking/test_sqlalchemy_store.py`